### DISCUSSION TITLE
Migrating custom chart type from vue-chartjs to vue-chart-3 (typescript + composition api)

### DISCUSSION THREAD

**Discussion by Shujee** [victorgarciaesgi/vue-chart-3#44] [Q&A]
Is there any example available for extending existing chart types in vue-chart-3? I have created a custom chart before using vue-chartjs, but that was with pure JS and vue 2. Now I want to migrate this chart to using TypeScript and Composition API (while still using Vue 2.x).

I need to override draw() function as I need to do some custom drawing. Any tips or a bare-bone example of extending the built-in bubble chart type would give me enough to start.

**Comment by victorgarciaesgi** [comment]
Typescript still can't let you alter existing types, only augment them (Like adding properties) :(
